GoogleForCreators/web-stories-wp
# Taxonomies: support custom taxonomies

## Context



## Summary

Adds some custom taxonomies that existed before (color, verticals). Playing with the labeling and the set up of these custom taxonomies.

The new taxonomies are prefixed with `web_story_`.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. run `composer install` to set up the new taxonomies
2. Run the editor. Should see the new taxonomies on a network request where the story is being fetched under `_links -> wp:term`:

![image](https://user-images.githubusercontent.com/22185279/135190854-c89ac9b3-9e42-4d91-b10e-7f84a6aa3826.png)



## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #
